src/Settings.h

```diff
@@ -1,24 +1,29 @@
 #pragma once
-#include "cpprest/http_client.h"
+
 #include <fstream>
 #include <iostream>
 
-namespace weathersvr {
+#include <cpprest/http_client.h>
+
+namespace weatherserver {
 
     class Settings {
 
         public:
 
             Settings();
+
             /*
-            Open the file passed to the formal parameter fileName and set the settings to variables of this class.
-            @param char* fileName - the path containing the settings file name and extension that is passed to command line arguments.
+              Assign default values to variables.
+              Attempt to open SETTINGS_FILE_NAME to get Dark Sky API key and other settings to override dafault values.
             */
-            bool setup(char* fileName);
+            Settings(std::string& currentDir);
+
+            const bool Settings::isValid() const { return settingsAreValid; }
```

Comment:
`const bool` is not required, just return `bool`
Also we can rename it to `areValid()` to be more consistent with member variable name.

Reply:
Done.